geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/command/GetAll70Test.java

```diff
@@ -157,18 +157,8 @@ public void integratedSecurityShouldFailIfNotAuthorized() throws Exception {
 
     this.getAll70.cmdExecute(this.message, this.serverConnection, this.securityService, 0);
 
-    for (Object key : KEYS) {
```

Comment:
What was your reason for this change? It seems the test is now much more lenient.